examples/rbfe_edge_list.py

```diff
@@ -97,54 +25,43 @@ def read_from_args():
     parser.add_argument("--n_gpus", type=int, help="number of gpus", required=True)
     parser.add_argument("--seed", type=int, help="random seed for the runs", required=True)
 
-    args = parser.parse_args()
-
-    mols = read_sdf(str(args.ligands))
-
-    cpc = CUDAPoolClient(args.n_gpus)
-    cpc.verify()
-
-    forcefield = Forcefield.load_from_file(args.forcefield)
-    protein = app.PDBFile(args.protein)
-
-    with open(args.results_csv) as csvfile:
-        reader = csv.reader(csvfile, delimiter=",")
-        next(reader)
-        rows = [row for row in reader]
-        futures = []
-        for row_idx, row in enumerate(rows):
-            mol_a_name, mol_b_name, exp_ddg, fep_ddg, fep_ddg_err, ccc_ddg, ccc_ddg_err = row
-            mol_a = get_mol_by_name(mols, mol_a_name)
-            mol_b = get_mol_by_name(mols, mol_b_name)
-
-            mcs_threshold = 2.0
-            core, smarts = atom_mapping.get_core_with_alignment(mol_a, mol_b, threshold=mcs_threshold)
-
-            print(f"Submitting job for {mol_a_name} -> {mol_b_name}")
-            futures.append(
-                cpc.submit(
-                    run_edge_and_save_results,
-                    mol_a,
-                    mol_b,
-                    core,
-                    forcefield,
-                    protein,
-                    args.n_frames,
-                    args.seed + row_idx,
-                    smarts,
-                    exp_ddg,
-                    fep_ddg,
-                    fep_ddg_err,
-                    ccc_ddg,
-                    ccc_ddg_err,
-                )
+    return parser.parse_args()
+
+
+def read_edges_csv(path: str) -> List[rbfe.Edge]:
+    with open(path) as fp:
+        reader = csv.reader(fp, delimiter=",")
+        next(reader, None)  # skip header
+        return [
+            rbfe.Edge(
+                mol_a_name,
+                mol_b_name,
+                {
+                    "exp_ddg_kcal": float(exp_ddg) * KCAL_TO_KJ,
+                    "fep_ddg_kcal": float(fep_ddg) * KCAL_TO_KJ,
+                    "fep_ddg_err_kcal": float(fep_ddg_err) * KCAL_TO_KJ,
+                    "ccc_ddg_kcal": float(ccc_ddg) * KCAL_TO_KJ,
+                    "ccc_ddg_err_kcal": float(ccc_ddg_err) * KCAL_TO_KJ,
```

Comment:
Hmm, good catch, this looks wrong. 1 = 1 kcal / 4.184 kJ and `KCAL_TO_KJ == 4.184`, so I think you're right that we should be dividing here.

Reply:
I think I just misunderstood the intended units of the output. I think it's best to remove the suffix from the outputs instead (since I think we generally follow the convention that no suffix means kJ)

Reply:
Addressed in f3c923f